src/AuthServiceProvider.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace EscolaLms\CsvUsers;
+
+use EscolaLms\Auth\Models\User;
+use EscolaLms\CsvUsers\Policies\CsvUsersPolicy;
+use Illuminate\Foundation\Support\Providers\AuthServiceProvider as ServiceProvider;
+use Laravel\Passport\Passport;
+
+class AuthServiceProvider extends ServiceProvider
+{
+    protected $policies = [
+        User::class => CsvUsersPolicy::class,
```

Comment:
This is issue with policies :( this will overwrite policy from `EscolaLms\Auth\Models\User` you cannot have multiple policies for one class. 

you should create you own class for User 

`src/Models/User.php` 

```php
namespace EscolaLms\CsvUsers\Models;

use EscolaLms\Auth\Models\User as AuthUser;

class User extends AuthUser {
}

```